src/style.css

```diff
@@ -0,0 +1,108 @@
+@import url(./stars.css);
+@import url(./reset.css);
+@import url(./vars.css);
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin: auto;
+  margin-bottom: 40px;
+}
+
+.card__name {
+  display: flex;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--primary-grey);
+  margin-bottom: 4px;
+}
+
+.card__info {
+  display: inline-flex;
+  align-self: flex-start;
+  flex-wrap: wrap;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-grey);
+  margin-bottom: 16px;
+}
+
+.card__review {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__review_number {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  color: var(--primary-grey);
+}
+
+.card__price {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price_text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-grey);
+}
+
+.card__price_value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--primary-grey);
+}
+
+.card__link {
+  display: flex;
+  background: var(--primary-blue);
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  cursor: pointer;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+}
+
+.card__link:hover {
+  background: #fff;
+  border: 1px solid var(--primary-blue);
```

Comment:
you can define border in `.card__link` directly and don’t need to change it on hover